internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,402 @@
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+type flagsImport struct {
+	overwrite               bool
+	kafkaApiKey             string
+	schemaRegistryApiKey    string
+	schemaRegistryApiSecret string
+}
+
+type BindingsXConfigs struct {
+	CleanupPolicy                  string `yaml:"cleanup.policy"`
+	DeleteRetentionMs              int    `yaml:"delete.retention.ms"`
+	ConfluentValueSchemaValidation string `yaml:"confluent.value.schema.validation"`
+}
+
+type KafkaBinding struct {
+	XPartitions int              `yaml:"x-partitions"`
+	XReplicas   int              `yaml:"x-replicas"`
+	XConfigs    BindingsXConfigs `yaml:"x-configs"`
+}
+
+type Message struct {
+	SchemaFormat string      `yaml:"schemaFormat"`
+	ContentType  string      `yaml:"contentType"`
+	Payload      interface{} `yaml:"payload"`
+	Name         string      `yaml:"name"`
+	Tags         []spec2.Tag `yaml:"tags"`
+}
+
+type Spec struct {
+	Asyncapi   string           `yaml:"asyncapi"`
+	Info       spec2.Info       `yaml:"info"`
+	Servers    spec2.Server     `yaml:"servers"`
+	Channels   map[string]Topic `yaml:"channels"`
+	Components Components       `yaml:"components"`
+}
+
+type Components struct {
+	Messages        map[string]Message `yaml:"messages"`
+	SecuritySchemes interface{}        `yaml:"securitySchemes"`
+}
+
+type Security struct {
+	ConfluentBroker         []interface{} `yaml:"confluentBroker"`
+	ConfluentSchemaRegistry []interface{} `yaml:"confluentSchemaRegistry"`
+}
+
+type MessageRef struct {
+	Ref string `yaml:"$ref"`
+}
+
+type Operation struct {
+	OperationID string                        `yaml:"operationId"`
+	TopicTags   []spec2.Tag                   `yaml:"tags"`
+	OpBindings  spec2.OperationBindingsObject `yaml:"bindings"`
+	Message     MessageRef                    `yaml:"message"`
+}
+
+type Topic struct {
+	Publish               Operation     `yaml:"publish"`
+	Subscribe             Operation     `yaml:"subscribe"`
+	XMessageCompatibility string        `yaml:"x-messageCompatibility"`
+	TopicBinding          TopicBindings `yaml:"bindings"`
+}
+
+type TopicBindings struct {
+	Kafka KafkaBinding `yaml:"kafka"`
+}
+
+type ConfluentBrokerXConfigs struct {
+	SaslMechanisms   string `yaml:"sasl.mechanisms"`
+	SaslPassword     string `yaml:"sasl.password"`
+	SaslUsername     string `yaml:"sasl.username"`
+	SecurityProtocol string `yaml:"security.protocol"`
+}
+type ConfluentBroker struct {
+	Type     string                  `yaml:"type"`
+	XConfigs ConfluentBrokerXConfigs `yaml:"x-configs"`
+}
+
+type ConfluentSRXConfigs struct {
+	BasicAuthUserInfo string `yaml:"basic.auth.user.info"`
+}
+type ConfluentSchemaRegistry struct {
+	Type     string              `yaml:"type"`
+	XConfigs ConfluentSRXConfigs `yaml:"x-configs"`
+}
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "import spec.yaml",
```

Comment:
This should not be a concrete value. It should be a descriptor (surrounded with `<>` since it is required).